tests/qos/test_qos_sai.py

```diff
@@ -1033,6 +1035,7 @@ def testQosSaiLossyQueueVoq(
         """
         if dutTestParams["basicParams"]["sonic_asic_type"] != "cisco-8000":
             pytest.skip("Lossy Queue Voq test is not supported")
+
```

Comment:
this change is not necessary in function.
and it will cause conflict with changes in PR https://github.com/sonic-net/sonic-mgmt/pull/8480